samples/local-service-registry/local-registry-client/src/main/java/org/apache/servicecomb/samples/localregistry/localregistryclient/LocalRegistryClient.java

```diff
@@ -30,10 +30,14 @@
   private static RestTemplate templateNew = RestTemplateBuilder.create();
 
   public static void main(String[] args) throws Exception {
-    System.setProperty("local.registry.file", "src/main/resources/registry.yaml");
+    String absoluteFilePath = Class.class.getClass().getResource("/").getPath() + "registry.yaml";
```

Comment:
better to support load based on classloader
current IMPL only can load from disk, can not load from jar resource

Reply:
change to use classloader to get absolute path